packages/@aws-cdk/aws-lambda/lib/code-signing-config.ts

```diff
@@ -0,0 +1,37 @@
+import { Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { CfnCodeSigningConfig } from './lambda.generated';
+
+export enum UntrustedArtifactOnDeployment {
+  ENFORCE = 'enforce',
+  WARN = 'warn',
+}
+
+export interface CodeSigningConfigOptions {
```

Comment:
This should be called `CodeSigningConfigProps`. Take a look at our design guidelines - https://github.com/aws/aws-cdk/blob/master/DESIGN_GUIDELINES.md